client/components/recurring-payment-icon/index.js

```diff
@@ -0,0 +1,64 @@
+import { __ } from '@wordpress/i18n';
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { Popover as PopoverComponent } from '@wordpress/components';
+import icon from './icon.svg';
+
+const Icon = styled.img`
+	height: 12px;
+	width: 12px;
+	cursor: pointer;
+`;
+
+const StyledPopover = styled( PopoverComponent )`
+	top: -11px !important;
+
+	.components-popover__content {
+		border-radius: 4px;
+		box-shadow: 0px 2px 6px 0px rgba( 0, 0, 0, 0.05 );
+		padding: 5px 10px;
+		background-color: #000000;
+		color: #ffffff;
+	}
```

Comment:
The styles here don't match what we're doing for the currency (i) popover. We should use the same styles or component.
![Screenshot 2025-05-13 at 13 56 17](https://github.com/user-attachments/assets/4fe4e86e-ce98-4d51-a0c0-5dba03aad2cc)

Reply:
Waiting on confirmation from design about this.

Reply:
As per the confirmation from design, we are keeping these different as one if a tooltip and another is a popover. However, in the previous implementation, I was using the popover component for showing `Supports recurring payments` message. I have updated the component in [facce12](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4299/commits/facce12b5604d9f84ce10561b5b120093df918ce)